### DISCUSSION TITLE
Install Root

### DISCUSSION THREAD

**Discussion by Zhaowei9908** [MadAnalysis/madanalysis5#36] [Q&A]
I was installing Delphes after installing root in madgraph.However,it still report this error
Command "install Delphes" interrupted with error:
InvalidCmd : In order to install ExRootAnalysis, you need to install Root on your computer first.
	please follow information on http://root.cern.ch/drupal/content/downloading-root

**Comment by jackaraz** [comment]
Hi, this is **MadAnalysis** discussion forum. Please refer to **MadGraph** authors if you have **MadGraph** related questions.

**Reply by Zhaowei9908** [reply]
sorry
